Lesson24/selcan1/index..html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+    <style>
+        .kapsayici{
+            width: 100%;;
+            height:100vh;
+            background-color: rgb(60, 60, 104); 
+            display: flex; 
+            justify-content: center;  
+            align-items: center;   
+          }
+        .anaMenu{
+            border-radius: 25px;
+            background-color: white;
+            width: 50%;
+            height: 50%;
+            padding: 30px;
+            display: flex;
+            flex-direction: row;  
+         }
+         @media all and (max-width: 576px) {
```

Comment:
Use a larger breakpoint, e.g. 768px as the page does not look very good on the devices between 576px and 768px.